test/userrepo-test.js

```diff
@@ -222,14 +228,14 @@ describe('User Repo', () => {
     });
 
     it('should isolate a user ID and its values of any relevant data', () => {
-      expect(userRepo.isolateUsernameAndRelevantData( 'sleepQuality', weeklySleepData)).to.eql({
+      expect(userRepo.isolateUsernameAndRelevantData('sleepQuality', weeklySleepData)).to.eql({
         '1': [2.2],
         '2': [4.7],
         '3': [3]
       });
-      expect(userRepo.isolateUsernameAndRelevantData( 'numOunces', weeklyHydrationData)).to.eql({
-        '1': [ 37 ],
-        '2': [ 38 ] 
+      expect(userRepo.isolateUsernameAndRelevantData('numOunces', weeklyHydrationData)).to.eql({
+        '1': [37],
+        '2': [38]
       });
```

Comment:
Excellent job with the Calculations class! Maybe next time break it up into different branches?

Reply:
Yes I totally agree! It started as a refactor branch but then took a turn into a feature with a new class and also refactoring. Next time!